lib/cloudWatch.js

```diff
@@ -23,18 +24,58 @@ export default class CloudWatch {
     this.namespace = process.env.SERVICE_NAME || 'coinboss';
   }
 
-  putPriceMetric(args) {
-    return this.cloudwatch.putMetricData({
+  putMetricData(args) {
+    const data = {
       MetricData: [
+       {
+         MetricName: args.metricName,
+         Timestamp: moment().utc().format(),
```

Comment:
I’ve found the [documentation](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/CloudWatch.html#putMetricData-property) in case you’re interested @tvb, and it indeed takes a `new Date`. I was surprised that CloudWatch didn’t require ISO8601, but I guess it makes sense for the js SDK to be able to work with pure JS date.